### PR TITLE
HL-211: Summary page date formats

### PR DESCRIPTION
**Pull request by melniiv**
## Description :sparkles:
1. fixed date formats in applicant summary view
2. fixed date formats in handlers review page

The fields fixed:
deminimis aid granted at (in the listing table)
startdate
enddate

### CODE REVIEW

File: frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx [hunk 1]
```diff
@@ -128,13 +129,17 @@ const EmployeeView: React.FC<EmployeeViewProps> = ({
           <$ViewField>
             {t(`${translationsBase}.employee.fields.startDate.label`)}
           </$ViewField>
-          <$ViewField>{data.startDate ? data.startDate : '-'}</$ViewField>
+          <$ViewField>
+            {data.startDate ? formatDate(new Date(data.startDate)) : '-'}
```
Comment (nenonja) [inline frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx:133]: `{convertToUIDateFormat(data.startDate) || '-'}` should do the same?

File: frontend/benefit/handler/src/components/applicationReview/deminimisView/DeminimisView.tsx [hunk 2]
```diff
@@ -41,7 +41,11 @@ const DeminimisView: React.FC<ApplicationReviewViewProps> = ({ data }) => {
             </$SummaryTableHeader>
           </$GridCell>
           {data.deMinimisAidSet?.map((aid: DeMinimisAid) => (
-            <React.Fragment key={`${aid.granter ?? ''}${aid.grantedAt ?? ''}`}>
+            <React.Fragment
+              key={`${aid.granter ?? ''}${formatDate(
+                new Date(aid.grantedAt || '')
```
Comment (nenonja) [inline frontend/benefit/handler/src/components/applicationReview/deminimisView/DeminimisView.tsx:46]: `{convertToUIDateFormat(aid.grantedAt)}` should do the same? It returns empty string if not valid date or empty value.

**Review by nenonja — COMMENTED** [review]
one small suggestion added

File: frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx (hunk 1 above)
Comment (melniiv) [reply frontend/benefit/applicant/src/components/applications/forms/application/step5/employeeView/EmployeeView.tsx:133]: ok, didnt know that we have the helper, i ll fix

File: frontend/benefit/handler/src/components/applicationReview/deminimisView/DeminimisView.tsx (hunk 2 above)
Comment (melniiv) [reply frontend/benefit/handler/src/components/applicationReview/deminimisView/DeminimisView.tsx:46]: ok, didnt know that we have the helper, i ll fix

**Review by nenonja — APPROVED** [review]
Looks good and everything is green ✅

**Pull request merged**